full_bear/permissions.py

```diff
@@ -8,3 +9,11 @@ def has_permission(self, request: Request, view: APIView) -> bool:
         if request.method not in SAFE_METHODS:
             return True
         return bool(request.user and request.user.is_staff)
+
+
+class IsOwner(BasePermission):
+    def has_object_permission(self, request: Request, view: APIView, obj: Model) -> bool:
+        if not request.user:
+            return False
+        else:
+            return bool(request.user.is_staff or request.user == obj.user)
```

Comment:
```suggestion
        if not request.user:
            return False
        return request.user.is_staff or request.user == obj.user
```
No need for else and explicit type conversion to bool